### PR TITLE
Fix multiblock on ARM64EC

### PR DESCRIPTION
**Pull request by bylaws**

### CODE REVIEW

File: FEXCore/Source/Interface/Core/Frontend.cpp [hunk 1]
```diff
@@ -967,8 +967,14 @@ void Decoder::BranchTargetInMultiblockRange() {
     TargetRIP &= 0xFFFFFFFFU;
   }
 
-  // If the target RIP is within the symbol ranges then we are golden
-  if (TargetRIP >= SymbolMinAddress && TargetRIP < SymbolMaxAddress) {
+  // If the target RIP is x86 code within the symbol ranges then we are golden
+  bool ValidMultiblockMember = TargetRIP >= SymbolMinAddress && TargetRIP < SymbolMaxAddress;
+
+#ifdef _M_ARM_64EC
+  ValidMultiblockMember = ValidMultiblockMember && !RtlIsEcCode(TargetRIP);
```
Comment (alyssarosenzweig) [inline FEXCore/Source/Interface/Core/Frontend.cpp:974]: Nit: `ValidMultiblockMember &= !RtlIsEcCode(TargetRIP);`

**Review by alyssarosenzweig — APPROVED** [review]
+1 with the nit fixed

File: FEXCore/Source/Interface/Core/Frontend.cpp (hunk 1 above)
Comment (bylaws) [reply FEXCore/Source/Interface/Core/Frontend.cpp:974]: ah sure, suppose the ec check is cheap enough the lack of short circuiting doesn't really matter

Comment (alyssarosenzweig) [reply FEXCore/Source/Interface/Core/Frontend.cpp:974]: Oh.. right.. short circuiting might help perf here... mm, I guess this is ok as is then.

**Review by alyssarosenzweig — APPROVED** [review]

**Pull request merged**